### DISCUSSION TITLE
mdn-http-observatory standalone API with postgres not reading conf file or trouble with getting the config right

### DISCUSSION THREAD

**Discussion by spundae** [mdn/mdn-community#767] [Content]
I am working on a local API infrastructure with postgres. I read that I need to rename the config file:
Create a configuration file by copying the [config/config-example.json](https://github.com/mdn/mdn-http-observatory/blob/main/conf/config-example.json) to config/config.json. Put in your database credentials into config/config.json

I filled in the database credentials and placed it into ./conf/config.json. That didn;t work and I placed it into ./config/config.json in the observatory directory which I created because it was not there.
I might be newbie but even tried strace to figure out if it is looking for that config file but it isn't.

Does somebody has this local observatory API running and can point me in the right direction how to solve my error which is:

> @mdn/mdn-http-observatory@1.3.7 migrate
> node -e 'import("./src/database/migrate.js").then( m => m.migrateDatabase() )'

Error: SASL: SCRAM-SERVER-FIRST-MESSAGE: client password must be a string
    at /usr/local/lib/node_modules/@mdn/mdn-http-observatory/node_modules/pg-pool/index.js:45:11
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async PostgresClient.ensureTable (file:///usr/local/lib/node_modules/@mdn/mdn-http-observatory/node_modules/postgrator/lib/Client.js:93:21)
    at async Postgrator.migrate (file:///usr/local/lib/node_modules/@mdn/mdn-http-observatory/node_modules/postgrator/postgrator.js:256:7)
    at async Module.migrateDatabase (file:///usr/local/lib/node_modules/@mdn/mdn-http-observatory/src/database/migrate.js:38:32) {
  appliedMigrations: []
}

I don't know how I can made it more STRING, because it says string in the config and is between " ".

Thank you in advance

**Comment by spundae** [comment]
Managed to solve this myself

**Reply by dipikabh** [reply]
Thanks for letting us know.
